src/components/calcite-color-picker/calcite-color-picker.tsx

```diff
@@ -462,22 +506,53 @@ export class CalciteColorPicker {
     } = this;
     const selectedColorInHex = color ? color.hex() : null;
     const hexInputScale = scale !== "s" ? "m" : scale;
-    const { colorFieldAndSliderInteractive } = this;
+    const {
+      colorFieldAndSliderInteractive,
+      scopeTop,
+      scopeLeft,
+      hueScopeLeft,
+      hueScopeTop,
+      scopeOrientation
+    } = this;
     const elementDir = getElementDir(el);
     const noColor = color === null;
-
+    const vertical = scopeOrientation === "vertical";
     return (
       <div class={CSS.container}>
-        <canvas
-          class={{
-            [CSS.colorFieldAndSlider]: true,
-            [CSS.colorFieldAndSliderInteractive]: colorFieldAndSliderInteractive
-          }}
-          onMouseEnter={this.handleColorFieldAndSliderMouseEnterOrMove}
-          onMouseLeave={this.handleColorFieldAndSliderMouseLeave}
-          onMouseMove={this.handleColorFieldAndSliderMouseEnterOrMove}
-          ref={this.initColorFieldAndSlider}
-        />
+        <div class={CSS.colorFieldAndSliderWrap}>
+          <canvas
+            class={{
+              [CSS.colorFieldAndSlider]: true,
+              [CSS.colorFieldAndSliderInteractive]: colorFieldAndSliderInteractive
+            }}
+            onMouseEnter={this.handleColorFieldAndSliderMouseEnterOrMove}
+            onMouseLeave={this.handleColorFieldAndSliderMouseLeave}
+            onMouseMove={this.handleColorFieldAndSliderMouseEnterOrMove}
+            ref={this.initColorFieldAndSlider}
+          />
+          <div
+            aria-label={vertical ? this.intlValue : this.intlSaturation}
```

Comment:
Should this slider's label be for value **and** saturation? I'm wondering what the best approach would be here since the screenreader would give no context about the value type changing based on direction.

Reply:
We are in uncharted waters here. The spec doesn't have a two-dimensional slider. Most of the responses I found on forums basically said "just ignore this for screen readers", which I can't do because the whole point is that it's in the tab order, so if you can tab to it you have to allow screen reader access. 

Probably creating a component that controls two independent sliders would be better but man I have no idea how you'd even do that and there are 0 examples of that kind of thing...

Reply:
I think we can make the accessibility of the labeling better in a future pr.